CHANGELOG.md

```diff
@@ -3,6 +3,25 @@
 # Changelog
 All notable changes to this project are documented in this file.
 
+## 1.9.0 - Unreleased
```

Comment:
Hmm, unrelated? :) Some of those items will hopefully be in 1.8.1, instead of 1.9.0?

Reply:
I just wanted to not forget to add the MSRV in there so I did fill all entries.
But yes, when we cherry pick item we'll have to edit the changelog to remove the item which went in 1.8.1 if we do release a 1.8.1